target/linux/ramips/dts/mt7621_snr_snr-cpe-me2-lite.dts

```diff
@@ -0,0 +1,164 @@
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "snr,snr-cpe-me2-lite", "mediatek,mt7621-soc";
+	model = "SNR-CPE-ME2-Lite";
+
+	leds {
```

Comment:
I recommend describing LED as fresh Linux kernels.
https://www.kernel.org/doc/html/latest/leds/leds-class.html
Example:
https://github.com/openwrt/openwrt/blob/f1c9afd801380a05a91d979b475c76cc0a67caae/target/linux/ramips/dts/mt7621_iptime_t5004.dts#L7
But this recommendation is not binding.